tests/test_iotedgemodule.py

```diff
@@ -0,0 +1,34 @@
+import json
+import logging
+import os
+import pytest
+
+root_dir = os.getcwd()
+module_name = "iotedgeFilterModule"
+file_list = ["requirements.txt", "module.json", "main.py", "Dockerfile.arm32v7",
+             "Dockerfile.amd64.debug", "Dockerfile.amd64", ".gitignore"]
+image_repository="localhost:5000/"+module_name
+
+
+@pytest.fixture(scope="module")
+def test_install_iotedgemodule(request):
```

Comment:
Please clean up the generated directory (see https://docs.pytest.org/en/latest/fixture.html#fixture-finalization-executing-teardown-code)

Reply:
Minor naming suggestion: we are actually installing the module, but do scaffolding, so `scaffold` should be a better name.

Reply:
updated